src/mkdocs_bibtex/utils.py

```diff
@@ -168,3 +169,21 @@ def format_bibliography(citation_quads):
         bibliography.append(bibliography_text)
 
     return "\n".join(bibliography)
+
+
+def tempfile_from_url(url, suffix='.bib'):
```

Comment:
Remove this default value since you don't predominantly use it with the '.bib' extension. 
```suggestion
def tempfile_from_url(url, suffix):
```